shared/chat/conversation/info-panel/index.desktop.js

```diff
@@ -1,19 +1,19 @@
 // @flow
 import * as React from 'react'
-import {Box, Button, Checkbox, Divider, Icon} from '../../../common-adapters'
+import {Avatar, Box, Button, Checkbox, Divider, Icon, Text} from '../../../common-adapters'
 import {globalColors, globalMargins, globalStyles} from '../../../styles'
 import Participants from './participants'
 
-import type {Props} from '.'
+import type {InfoPanelProps, ChannelInfoPanelProps} from '.'
 
 const border = `1px solid ${globalColors.black_05}`
-const InfoPanel = (props: Props) => (
+const InfoPanel = (props: InfoPanelProps) => (
```

Comment:
small isn't about the number of participants. its that its scope is small (no channels or special rights). The ux shows 'Big teams' and there's implied 'Small' teams so thats why I chose the name. The thing that divides them is about the complexity and not the number of participants. ie you could have a 'big team' with 2 people and a 'small team' with 100 if you wanted